src/Uno.UWP/UI/Core/CoreDispatcher.netstd.cs

```diff
@@ -10,9 +10,17 @@ public sealed partial class CoreDispatcher
     {
 		private Timer _timer;
 
+		/// <summary>
+		/// Provide a action that will delegate the dispach of CoreDispatcher work
+		/// </summary>
+		public static Action<Action> DispatchOverride;
```

Comment:
hmm... maybe this should at least be marked EditorBrowsable never

Reply:
Sure, I'll add that.